tools/runme/build.sh

```diff
@@ -111,6 +111,21 @@ _sbt_build() {
   cd "$owd"
 }
 
+_sbt_adb_notebooks() {
+  show section "Running ADB Notebooks"
+  if [[ "$BUILDMODE" = "server" ]]; then
+    local token="$(__ az keyvault secret show --vault-name mmlspark-keys --name adb-token)"
+    token="${token##*\"value\": \"}"; token="${token%%\"*}"
+    export MML_ADB_TOKEN="$token"
+  fi
+
+  local owd="$PWD"
+  cd "$SRCDIR"
+  local TESTS="$TESTS"
+  _sbt_run "core-test-fuzzing/it:test"
```

Comment:
If `_sbt_run` is used from several places, it's better to make it do its own `cd "$SRCDIR"` and restore the WD when it's done.

Reply:
good idea